sound/soc/sof/imx/imx8.c

```diff
@@ -609,32 +609,32 @@ static struct snd_sof_dsp_ops sof_imx8x_ops = {
 };
 
 static struct sof_dev_desc sof_of_imx8qxp_desc = {
-	.ipc_supported_mask	= BIT(SOF_IPC),
-	.ipc_default		= SOF_IPC,
+	.ipc_supported_mask	= BIT(SOF_IPC_TYPE_3),
+	.ipc_default		= SOF_IPC_TYPE_3,
 	.default_fw_path = {
-		[SOF_IPC] = "imx/sof",
+		[SOF_IPC_TYPE_3] = "imx/sof",
```

Comment:
I wonder if the PR could be kept smaller by accepting that SOF_IPC continues to be used to refer to 3.

Reply:
I think it is more intuitive for the first look to have `SOF_IPC_TYPE_3` than `SOF_IPC`
IPC4 is SOF_IPC, it is IPC type used by SOF.